eslint.config.mjs

```diff
@@ -290,8 +290,7 @@ export default typescript.config([
       'no-cond-assign': ['error', 'always'],
       'no-case-declarations': 'off', // TODO(ryan953): Fix violations and delete this line
       'no-prototype-builtins': 'off', // TODO(ryan953): Fix violations and delete this line
-      'no-unsafe-optional-chaining': 'off', // TODO(ryan953): Fix violations and delete this line
-      'no-useless-escape': 'off', // TODO(ryan953): Fix violations and delete this line
+      'no-useless-escape': 'off',
```

Comment:
I looked at this one and I don't think it's worth fixing. Sometimes the extra escape char is helpful for reading a complex regexp. 

If someone doesn't agree we could still enable the rule though, i'm ok either way.